specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -156,6 +156,76 @@
           }
         ]
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers/{customerName}/providers/Microsoft.Subscription/createSubscription": {
+      "post": {
+        "description": "Creates an Azure subscription on behalf of a customer",
+        "operationId": "SubscriptionFactory_CustomerCreateSubscription",
+        "x-ms-examples": {
+          "createSubscription": {
+            "$ref": "./examples/customerCreateSubscription.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
```

Comment:
@jetts are you planning to send both headers for your POST operation? If yes, can I know the reason?
Why not use only one `Azure-AsyncOperation` header to track your long running operation?